packages/libs/react-ui/src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -25,18 +26,14 @@ const meta: Meta<
     docs: {
       description: {
         component:
-          'The Breadcrumb component displays the position of the current page within the site hierarchy, allowing page visitors to navigate the page hierarchy from their current location. It uses a composition of the `Root` and `Item` subcomponents to define the paths and structure of the entire breadcrumb component.<br><br><i>Note: In times when you need to use an external `Link` component (like next/link in Next.js), you can wrap the external component in `Breadcrumb.Item` and set the `asChild` prop to pass on styles and props to the child component.</i>',
+          'The Breadcrumb component displays the position of the current page within the site hierarchy, allowing page visitors to navigate the page hierarchy from their current location. It is composed by BreadcrumbsContainer and BreadcrumbsItem.<br><br><i>Note: In times when you need to use an external `Link` component (like next/link in Next.js), you can wrap the external component in BreadcrumbsItem and set the `asChild` prop to pass on styles and props to the child component.</i>',
       },
     },
   },
   argTypes: {
-    icon: {
+    startIcon: {
```

Comment:
I think it's better to keep the name `icon` since there will only ever be one icon. What do you think?

Reply:
I thought of this, but since we did it like that in Select for example I kept the same "idea" so it tells a little bit of the positioning, but i this idea, dont mind refactoring in both places

Reply:
Yeah, I think this one could go either way so I'm fine with this 👍

Reply:
imo usually bread crumbs have icons at the starting position only. So, its better to have it as `icon` when there is no possibility of having icon at the end.